packages/sql-graphql/index.js

```diff
@@ -118,7 +118,17 @@ async function mapperToGraphql (app, opts) {
     }
   }
 
-  let sdl = graphql.printSchema(new graphql.GraphQLSchema({ query, mutation, subscription }))
+  let sdl = ''
+  try {
+    sdl = graphql.printSchema(new graphql.GraphQLSchema({ query, mutation, subscription }))
+  } catch (error) {
+    // The next lines are excluded from test coverage:
+    // it's quite hard to test the following lines for all of the DB types
+    /* istanbul ignore next */
+    app.log.debug({ query, mutation, subscription }, 'GraphQL input schema')
+    /* istanbul ignore next */
+    throw new Error('Error printing the GraphQL schema: ' + error)
```

Comment:
```suggestion
    const newerr =  new Error('Error printing the GraphQL schema')
    newerr.cause = err
    throw newerr
```

Let's use the [`cause`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause) property instead.